### ISSUE TITLE
Printing ELL nodes error on re purposing  pretrained classifier

### ISSUE THREAD

**Issue by vishnoitanuj** [Microsoft/ELL#182]
input error: Error: couldn't read file: Failed to match field stride, instead found token 'extent'

**Comment by lovettchris** [comment]
I just tried it an it works fine, can you try again with the latest bits?
```
(ell) d:\Temp\tutorials\retarget>%ELL_root%\build\bin\release\print.exe -imap pretrained.ell --includeNodeId --refineIterations 1
<id:1528> InputNode<float>(12288)
<id:1529> ConstantNode<float>()
<id:1530> ConstantNode<float>()
<id:1531> BroadcastLinearFunctionNode<float>(1528.output[0:12288], 1529.output[0:3], 1530.output[0:3])
<id:1532> ConstantNode<float>()
<id:1533> ConstantNode<float>()
<id:1534> BroadcastLinearFunctionNode<float>(1531.output[0:12288], 1532.output[0:3], 1533.output[0:0])
<id:1535> ConstantNode<float>()
<id:1536> ConstantNode<float>()
<id:1537> BroadcastLinearFunctionNode<float>(1534.output[0:12288], 1535.output[0:0], 1536.output[0:3])
<id:1538> ReorderDataNode<float>(1537.output[0:13068])
<id:1539> UnrolledConvolutionNode<float>(1538.output[0:13068])
<id:1540> ReorderDataNode<float>(1539.output[0:131072])
<id:1541> ConstantNode<float>()
<id:1542> ConstantNode<float>()
<id:1543> BroadcastLinearFunctionNode<float>(1540.output[0:131072], 1541.output[0:0], 1542.output[0:32])
<id:1544> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1543.output[0:131072])
<id:1545> ConstantNode<float>()
<id:1546> ConstantNode<float>()
<id:1547> BroadcastLinearFunctionNode<float>(1544.output[0:131072], 1545.output[0:32], 1546.output[0:32])
<id:1548> ConstantNode<float>()
<id:1549> ConstantNode<float>()
<id:1550> BroadcastLinearFunctionNode<float>(1547.output[0:131072], 1548.output[0:32], 1549.output[0:0])
<id:1551> ConstantNode<float>()
<id:1552> ConstantNode<float>()
<id:1553> BroadcastLinearFunctionNode<float>(1550.output[0:131072], 1551.output[0:0], 1552.output[0:32])
<id:1554> ReorderDataNode<float>(1553.output[0:139392])
<id:1555> UnrolledConvolutionNode<float>(1554.output[0:139392])
<id:1556> ReorderDataNode<float>(1555.output[0:131072])
<id:1557> ConstantNode<float>()
<id:1558> ConstantNode<float>()
<id:1559> BroadcastLinearFunctionNode<float>(1556.output[0:131072], 1557.output[0:0], 1558.output[0:32])
<id:1560> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1559.output[0:131072])
<id:1561> PoolingLayerNode<float,MaxPoolingFunction>(1560.output[0:131072])
    PoolingLayer<float,MaxPoolingFunction>(shape=[64,64,32]->[32,32,32], function=maxpooling, stride=2, size=2)
<id:1562> ConstantNode<float>()
<id:1563> ConstantNode<float>()
<id:1564> BroadcastLinearFunctionNode<float>(1561.output[0:32768], 1562.output[0:32], 1563.output[0:32])
<id:1565> ConstantNode<float>()
<id:1566> ConstantNode<float>()
<id:1567> BroadcastLinearFunctionNode<float>(1564.output[0:32768], 1565.output[0:32], 1566.output[0:0])
<id:1568> ConstantNode<float>()
<id:1569> ConstantNode<float>()
<id:1570> BroadcastLinearFunctionNode<float>(1567.output[0:32768], 1568.output[0:0], 1569.output[0:32])
<id:1571> ReorderDataNode<float>(1570.output[0:36992])
<id:1572> UnrolledConvolutionNode<float>(1571.output[0:36992])
<id:1573> ReorderDataNode<float>(1572.output[0:65536])
<id:1574> ConstantNode<float>()
<id:1575> ConstantNode<float>()
<id:1576> BroadcastLinearFunctionNode<float>(1573.output[0:65536], 1574.output[0:0], 1575.output[0:64])
<id:1577> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1576.output[0:65536])
<id:1578> ConstantNode<float>()
<id:1579> ConstantNode<float>()
<id:1580> BroadcastLinearFunctionNode<float>(1577.output[0:65536], 1578.output[0:64], 1579.output[0:64])
<id:1581> ConstantNode<float>()
<id:1582> ConstantNode<float>()
<id:1583> BroadcastLinearFunctionNode<float>(1580.output[0:65536], 1581.output[0:64], 1582.output[0:0])
<id:1584> ConstantNode<float>()
<id:1585> ConstantNode<float>()
<id:1586> BroadcastLinearFunctionNode<float>(1583.output[0:65536], 1584.output[0:0], 1585.output[0:64])
<id:1587> ReorderDataNode<float>(1586.output[0:73984])
<id:1588> UnrolledConvolutionNode<float>(1587.output[0:73984])
<id:1589> ReorderDataNode<float>(1588.output[0:65536])
<id:1590> ConstantNode<float>()
<id:1591> ConstantNode<float>()
<id:1592> BroadcastLinearFunctionNode<float>(1589.output[0:65536], 1590.output[0:0], 1591.output[0:64])
<id:1593> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1592.output[0:65536])
<id:1594> PoolingLayerNode<float,MaxPoolingFunction>(1593.output[0:65536])
    PoolingLayer<float,MaxPoolingFunction>(shape=[32,32,64]->[16,16,64], function=maxpooling, stride=2, size=2)
<id:1595> ConstantNode<float>()
<id:1596> ConstantNode<float>()
<id:1597> BroadcastLinearFunctionNode<float>(1594.output[0:16384], 1595.output[0:64], 1596.output[0:64])
<id:1598> ConstantNode<float>()
<id:1599> ConstantNode<float>()
<id:1600> BroadcastLinearFunctionNode<float>(1597.output[0:16384], 1598.output[0:64], 1599.output[0:0])
<id:1601> ConstantNode<float>()
<id:1602> ConstantNode<float>()
<id:1603> BroadcastLinearFunctionNode<float>(1600.output[0:16384], 1601.output[0:0], 1602.output[0:64])
<id:1604> ReorderDataNode<float>(1603.output[0:20736])
<id:1605> UnrolledConvolutionNode<float>(1604.output[0:20736])
<id:1606> ReorderDataNode<float>(1605.output[0:32768])
<id:1607> ConstantNode<float>()
<id:1608> ConstantNode<float>()
<id:1609> BroadcastLinearFunctionNode<float>(1606.output[0:32768], 1607.output[0:0], 1608.output[0:128])
<id:1610> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1609.output[0:32768])
<id:1611> ConstantNode<float>()
<id:1612> ConstantNode<float>()
<id:1613> BroadcastLinearFunctionNode<float>(1610.output[0:32768], 1611.output[0:128], 1612.output[0:128])
<id:1614> ConstantNode<float>()
<id:1615> ConstantNode<float>()
<id:1616> BroadcastLinearFunctionNode<float>(1613.output[0:32768], 1614.output[0:128], 1615.output[0:0])
<id:1617> ConstantNode<float>()
<id:1618> ConstantNode<float>()
<id:1619> BroadcastLinearFunctionNode<float>(1616.output[0:32768], 1617.output[0:0], 1618.output[0:128])
<id:1620> ReorderDataNode<float>(1619.output[0:41472])
<id:1621> UnrolledConvolutionNode<float>(1620.output[0:41472])
<id:1622> ReorderDataNode<float>(1621.output[0:32768])
<id:1623> ConstantNode<float>()
<id:1624> ConstantNode<float>()
<id:1625> BroadcastLinearFunctionNode<float>(1622.output[0:32768], 1623.output[0:0], 1624.output[0:128])
<id:1626> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1625.output[0:32768])
<id:1627> PoolingLayerNode<float,MaxPoolingFunction>(1626.output[0:32768])
    PoolingLayer<float,MaxPoolingFunction>(shape=[16,16,128]->[8,8,128], function=maxpooling, stride=2, size=2)
<id:1628> ConstantNode<float>()
<id:1629> ConstantNode<float>()
<id:1630> BroadcastLinearFunctionNode<float>(1627.output[0:8192], 1628.output[0:128], 1629.output[0:128])
<id:1631> ConstantNode<float>()
<id:1632> ConstantNode<float>()
<id:1633> BroadcastLinearFunctionNode<float>(1630.output[0:8192], 1631.output[0:128], 1632.output[0:0])
<id:1634> ConstantNode<float>()
<id:1635> ConstantNode<float>()
<id:1636> BroadcastLinearFunctionNode<float>(1633.output[0:8192], 1634.output[0:0], 1635.output[0:128])
<id:1637> ReorderDataNode<float>(1636.output[0:12800])
<id:1638> UnrolledConvolutionNode<float>(1637.output[0:12800])
<id:1639> ReorderDataNode<float>(1638.output[0:16384])
<id:1640> ConstantNode<float>()
<id:1641> ConstantNode<float>()
<id:1642> BroadcastLinearFunctionNode<float>(1639.output[0:16384], 1640.output[0:0], 1641.output[0:256])
<id:1643> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1642.output[0:16384])
<id:1644> PoolingLayerNode<float,MaxPoolingFunction>(1643.output[0:16384])
    PoolingLayer<float,MaxPoolingFunction>(shape=[8,8,256]->[4,4,256], function=maxpooling, stride=2, size=2)
<id:1645> ConstantNode<float>()
<id:1646> ConstantNode<float>()
<id:1647> BroadcastLinearFunctionNode<float>(1644.output[0:4096], 1645.output[0:256], 1646.output[0:256])
<id:1648> ConstantNode<float>()
<id:1649> ConstantNode<float>()
<id:1650> BroadcastLinearFunctionNode<float>(1647.output[0:4096], 1648.output[0:256], 1649.output[0:0])
<id:1651> ConstantNode<float>()
<id:1652> ConstantNode<float>()
<id:1653> BroadcastLinearFunctionNode<float>(1650.output[0:4096], 1651.output[0:0], 1652.output[0:256])
<id:1654> ReorderDataNode<float>(1653.output[0:9216])
<id:1655> UnrolledConvolutionNode<float>(1654.output[0:9216])
<id:1656> ReorderDataNode<float>(1655.output[0:8192])
<id:1657> ConstantNode<float>()
<id:1658> ConstantNode<float>()
<id:1659> BroadcastLinearFunctionNode<float>(1656.output[0:8192], 1657.output[0:0], 1658.output[0:512])
<id:1660> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1659.output[0:8192])
<id:1661> PoolingLayerNode<float,MaxPoolingFunction>(1660.output[0:8192])
    PoolingLayer<float,MaxPoolingFunction>(shape=[4,4,512]->[2,2,512], function=maxpooling, stride=2, size=2)
<id:1662> ReorderDataNode<float>(1661.output[0:2048])
<id:1663> UnrolledConvolutionNode<float>(1662.output[0:2048])
<id:1664> ReorderDataNode<float>(1663.output[0:4000])
<id:1665> ConstantNode<float>()
<id:1666> ConstantNode<float>()
<id:1667> BroadcastLinearFunctionNode<float>(1664.output[0:4000], 1665.output[0:0], 1666.output[0:1000])
<id:1668> BroadcastUnaryFunctionNode<float,ReLUActivationFunction<float>>(1667.output[0:4000])
<id:1669> PoolingLayerNode<float,MeanPoolingFunction>(1668.output[0:4000])
    PoolingLayer<float,MeanPoolingFunction>(shape=[2,2,1000]->[1,1,1000], function=meanpooling, stride=1, size=2)
<id:1670> SoftmaxLayerNode<float>(1669.output[0:1000])
    SoftmaxLayer<float>(shape=[1,1,1000]->[1,1,1000])
<id:1671> OutputNode<float>(1670.output[0:1000])
```
```

**Comment by kernhanda** [comment]
Fixed with release [v2.3.7](https://github.com/Microsoft/ELL/releases/tag/v2.3.7).
